blazor/diagram/tool-tip.md

```diff
@@ -19,4 +19,656 @@ By default, the diagram displays a tooltip that provides information about the s
+@using Syncfusion.Blazor.Popups
+<input type="button" value="Show CustomTooltip" @onclick="show" />
+<input type="button" value="Hide CustomTooltip" @onclick="hide" />
+<SfDiagramComponent @ref="@diagram" Width="1000px" Height="500px" Nodes="@nodes" />
+@code
+{
+    DiagramObjectCollection<Node> nodes;
+    SfDiagramComponent diagram;
+    protected override void OnInitialized()
+    {
+        nodes = new DiagramObjectCollection<Node>();
+        Node node = new Node()
+        {
+            ID = "node1",
+            OffsetX = 100,
+            OffsetY = 100,
+            Width = 100,
+            Height = 100,
+            Style = new ShapeStyle() 
+            { 
+                Fill = "#6495ED", 
+                StrokeColor = "white" 
+            },
+            Tooltip = new DiagramTooltip(){Content="Auto",OpensOn="Auto",Position=Position.BottomCenter},
+            Constraints = NodeConstraints.Default|NodeConstraints.Tooltip,
+        };
+        Node node1 = new Node()
+        {
+            ID = "node2",
+            OffsetX = 250,
+            OffsetY = 100,
+            Width = 100,
+            Height = 100,
+            Style = new ShapeStyle() 
+            { 
+                Fill = "#6495ED", 
+                StrokeColor = "white" 
+            },
+            Tooltip = new DiagramTooltip(){Content="Hover",OpensOn="Hover"},
+            Constraints = NodeConstraints.Default|NodeConstraints.Tooltip,
+        };
+        Node node2 = new Node()
+        {
+            ID = "node3",
+            OffsetX = 400,
+            OffsetY = 100,
+            Width = 100,
+            Height = 100,
+            Style = new ShapeStyle() 
+            { 
+                Fill = "#6495ED", 
+                StrokeColor = "white" 
+            },
+            Tooltip = new DiagramTooltip(){Content="Custom",OpensOn="Custom"},
+            Constraints = NodeConstraints.Default|NodeConstraints.Tooltip,
+        };
+        Node node3 = new Node()
+        {
+            ID = "node4",
+            OffsetX = 550,
+            OffsetY = 100,
+            Width = 100,
+            Height = 100,
+            Style = new ShapeStyle() 
+            { 
+                Fill = "#6495ED", 
+                StrokeColor = "white" 
+            },
+            Tooltip = new DiagramTooltip() { Content = "Click", OpensOn = "Click" },
+            Constraints = NodeConstraints.Default|NodeConstraints.Tooltip,
+        };
+        nodes.Add(node);
+        nodes.Add(node1);
+        nodes.Add(node2);
+        nodes.Add(node3);
+    }
+   
+    private  void show()
+    {
+        diagram.ShowTooltip(diagram.Nodes[2] as NodeBase);
+    }
+    private  void hide()
+    {
+       diagram.HideTooltip(diagram.Nodes[2] as NodeBase);
+    }
+}
+```
+|![ToolTip During hover the node with open mode](images/blazor-diagram-openmodenodetooltip.gif) | 
+
+The following code example illustrates how to set open mode to the tooltip for connectors.
```

Comment:
The following code example illustrates how to set the open mode to the tooltip for the connectors.